site/content/docs/user/pod-security-policy.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Pod Security Policy"
+menu:
+  main:
+    parent: "user"
+    identifier: "pod-security-policy"
+    weight: 3
+---
+
+# Pod Security Policy
+
+This guide covers setting up [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
+on a kind cluster.
+
+## Enabling Pod Security Policy
+
+Pod security policy control is implemented as an optional (but recommended) admission controller. PodSecurityPolicies are enforced by enabling the admission controller, but doing so without authorizing any policies will prevent any pods from being created in the cluster.
```

Comment:
i'd add a note here that PSP *might be going away*. not sure where the slides are, but you can link to the sig-auth agenda document - e.g. "SIG Auth is having discussions about ...."